ytypes/schema_tests/validate_test.go

```diff
@@ -800,12 +802,12 @@ func TestLeafrefCurrent(t *testing.T) {
 		t.Fatalf("TestLeafrefCurrent: could not create subinterface, got: %v, want error: nil", err)
 	}
 
-	ygot.BuildEmptyTree(dev)
-	mi, err := dev.Mpls.Global.NewInterface("eth0.0")
+	ygot.BuildEmptyTree(ni)
+	mi, err := ni.Mpls.Global.NewInterface("eth0.0")
 	if err != nil {
 		t.Fatalf("TestLeafrefCurrent: could not add new MPLS interface, got: %v, want error: nil", err)
 	}
-	mi.InterfaceRef = &oc.Mpls_Global_Interface_InterfaceRef{
+	mi.InterfaceRef = &oc.NetworkInstance_Mpls_Global_Interface_InterfaceRef{
```

Comment:
Also unsure if this is related.

Reply:
Per other change, this is a schema change (we removed /mpls).